src/test/java/HomeWork/DropdownList.java

```diff
@@ -0,0 +1,22 @@
+package HomeWork;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.util.List;
+
+public class DropdownList extends PreCondition {
+    @Test
+    public void dropdownList() {
+        driver.navigate().to("https://the-internet.herokuapp.com/dropdown");
+        List<WebElement> Rows = driver.findElements(By.cssSelector("#dropdown > option"));
```

Comment:
it's absolutelly incorrect. According to the task you must use "Select" class-decorator to work with dropdown